src/site/content/en/blog/user-preference-media-features-headers/index.md

```diff
@@ -0,0 +1,186 @@
+| `Sec-CH-Forced-Colors`                | [`active`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-forced-colors-active), [`none`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-forced-colors-none)                                                                                                                                                                                                       | [`forced-colors`](https://drafts.csswg.org/mediaqueries-5/#forced-colors)                               |
+| `Sec-CH-Prefers-Color-Scheme`         | [`light`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-color-scheme-light), [`dark`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-color-scheme-dark)                                                                                                                                                                                           | [`prefers-color-scheme`](https://drafts.csswg.org/mediaqueries-5/#prefers-color-scheme)                 |
+| `Sec-CH-Prefers-Reduced-Data`         | [`no-preference`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-reduced-data-no-preference), [`reduce`](https://drafts.csswg.org/mediaqueries-5/#valdef-media-prefers-reduced-data-reduce)                                                                                                                                                                       | [`prefers-reduced-data`](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-data)                 |
+
+## Browser support
+
+The `Sec-CH-Prefers-Color-Scheme` client hint header is supported on Chromium&nbsp;93.
+Other vendors' feedback, namely [WebKit's](https://lists.webkit.org/pipermail/webkit-dev/2021-May/031856.html)
+and [Mozilla's](https://github.com/mozilla/standards-positions/issues/526), is pending.
+
+## Demo of `Sec-CH-Prefers-Color-Scheme`
+
+Try the [demo](https://sec-ch-prefers-color-scheme.glitch.me/) in Chromium&nbsp;93 and notice how
+the inlined CSS changes according to the user's preferred color scheme.
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/pKAKyrN18CjhAYUNpJyk.png", alt="Sec-CH-Prefers-Color-Scheme: dark", width="800", height="541" %}
+
+{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/6Xujcgyveo0QY0E3LQOF.png", alt="Sec-CH-Prefers-Color-Scheme: light", width="800", height="541" %}
+
+## How it works
+
+1. The client makes an initial request to the server.
+   ```bash
+   GET / HTTP/2
+   Host: example.com
+   ```
+1. The server responds, telling the client via `Accept-CH` that it accepts the
+   `Sec-CH-Prefers-Color-Scheme` and the `Sec-CH-Prefers-Contrast` Client Hints, out of which as per
+   `Critical-CH` it considers `Sec-CH-Prefers-Color-Scheme` a Critical Client Hint that it also
+   varies the response on as conveyed by `Vary`.
+   ```bash
+   HTTP/1.1 200 OK
```

Comment:
I'm fine leaving it as it as HTTP/2 is a binary protocol but I wanted to catch your attention on this.

Reply:
Thanks. Updated to `/2` everywhere now.